pkg/kritis/cryptolib/verifier.go

```diff
@@ -137,10 +152,16 @@ func (v *verifier) VerifyAttestation(att *Attestation) error {
 		return err
 	}
 
+	// TODO(https://github.com/grafeas/kritis/issues/503): Determine whose
+	// responsibility it is to check the payload. If cryptolib is responsible
+	// determine an API for checking the payload.
 	// Extract the payload into an AuthenticatedAttestation, whose contents we
 	// can trust.
-	actual := formAuthenticatedAttestation(payload)
-	return v.checkAuthenticatedAttestation(actual, v.ImageDigest)
+	authenticatedAtt, err := v.formAuthenticatedAttestation(payload)
```

Comment:
If this is only used for `checkAuthenticatedAttestation`. Can we do the `form` in `checkAuthenticatedAttestation`? It makes the verifier struct leaner and also centralizes user-specific payload operation into one function.

Reply:
Done. Conversion is now done completely within `checkAuthenticatedAttestation`